src/read/coff/import.rs

```diff
@@ -0,0 +1,208 @@
+//! Support for reading short import files.
+//!
+//! These are used by some Windows linkers as a more compact way to describe
+//! dynamically imported symbols.
+
+use crate::read::{Architecture, Error, ReadError, ReadRef, Result};
+use crate::{pe, ByteString, Bytes, LittleEndian as LE};
+
+/// A Windows short form description of a symbol to import.
+/// Used in Windows import libraries. This is not an object file.
+#[derive(Debug, Clone)]
+pub struct MsImportFile<'data> {
+    header: &'data pe::ImportObjectHeader,
+    dll: ByteString<'data>,
+    symbol: ByteString<'data>,
+    kind: ImportType,
+    import: Option<ByteString<'data>>,
```

Comment:
It could but the reason I used an `Option` here is because `ByteString` makes for slightly better debug output. Whereas `ImportName` is a public enum so just uses a `&[u8]`.